src/internal/components/dropdown-footer/__tests__/dropdown-footer.test.tsx

```diff
@@ -31,8 +31,8 @@ describe('Dropdown footer', () => {
 
   test('adds correct aria attributes', () => {
     const { wrapper } = renderComponent(<DropdownFooter />);
-    const element = wrapper.find('div')!.getElement();
-    expect(element).toHaveAttribute('aria-live', 'polite');
-    expect(element).toHaveAttribute('aria-atomic', 'true');
+    const element = wrapper.find('span')!.getElement();
+    expect(element.firstElementChild).toHaveAttribute('aria-live', 'polite');
+    expect(element.firstElementChild).toHaveAttribute('aria-atomic', 'true');
```

Comment:
I don't mind having these tests, but how useful are they since we're using the shared component? we don't have this test for other components that uses `LiveRegion`
however we have test to check if the text is being updated

Reply:
The purpose of this test is not testing the structure of `LiveRegion` but to check if `aria-live` being used on the footer. No matter it is from `LiveRegion` or not.